.github/actions/use-local-dockerfile/action.yml

```diff
@@ -1,13 +1,14 @@
-name: "Setup Docker"
+name: "Use local Dockerfile"
 description: "Use the local Dockerfile to test the action instead of the one on DockerHub"
 
-runs: 
+runs:
   using: "composite"
   steps:
     - name: Replace image on action.yml
       shell: bash
       env:
-        ESC_DOCKER_IMAGE_LINK: 'docker\:\/\/sentryintegrations\/sentry-github-action-release\:latest'
-      run: | 
-        mv action.yml previous.yml 
+        ESC_DOCKER_IMAGE_LINK: 'docker\:\/\/ghcr.io\/getsentry\/action-release-image\:latest'
+      run: |
+        mv action.yml previous.yml
         sed "s/$ESC_DOCKER_IMAGE_LINK/Dockerfile/" previous.yml >> action.yml
```

Comment:
fun little sed trick -- you can use any character as the delimiter so you don't need the complex escapes

so you can change the `IMAGE_LINK` to be the literal value and use `sed "s|$ESC_DOCKER_IMAGE_LINK|Dockerfile/"`

Reply:
this can also be `sed -i` and then you can remove the `mv` commands (note this requires gnu sed so won't work on macos, but that's fine here!)